### PR TITLE
Remove app insights logging from tests

### PR DESCRIPTION
**Pull request by drwill-ms**
AppInsights logging is broken, we're not using it, and not missing it. So, let's remove it to reduce cost and time to run tests.

### CODE REVIEW

File: e2e/test/helpers/AmqpConnectionStatusChange.cs [hunk 1]
```diff
@@ -8,21 +8,19 @@ namespace Microsoft.Azure.Devices.E2ETests.Helpers
...
-            _logger = logger;
         }
 
         public int ConnectionStatusChangeCount { get; set; }
 
         public void ConnectionStatusChangeHandler(ConnectionStatusInfo connectionStatusInfo)
         {
             ConnectionStatusChangeCount++;
-            _logger.Trace($"{nameof(AmqpConnectionStatusChange)}.{nameof(ConnectionStatusChangeHandler)}: {_deviceId}: status={connectionStatusInfo.Status} statusChangeReason={connectionStatusInfo.ChangeReason} count={ConnectionStatusChangeCount}");
+            VerboseTestLogger.WriteLine($"{nameof(AmqpConnectionStatusChange)}.{nameof(ConnectionStatusChangeHandler)}: {_deviceId}: status={connectionStatusInfo.Status} statusChangeReason={connectionStatusInfo.ChangeReason} count={ConnectionStatusChangeCount}");
```
Comment (drwill-ms) [inline e2e/test/helpers/AmqpConnectionStatusChange.cs:23]: I figured I'd take the only logger remaining and replace existing traces with it.

Thoughts?

File: e2e/test/helpers/logging/VerboseTestLogger.cs [hunk 2]
```diff
@@ -3,33 +3,13 @@
...
-            EventSourceTestLogger.Log.TestVerboseMessage(message);
-            Debug.WriteLine(message);
-            Console.WriteLine(message);
-        }
-
-        public void WriteLine(string format, params object[] args)
+        public static void WriteLine(string message)
         {
-            string message = string.Format(CultureInfo.InvariantCulture, format, args);
             EventSourceTestLogger.Log.TestVerboseMessage(message);
```
[the remark this answers is not in the document]
Comment (drwill-ms) [reply e2e/test/helpers/logging/VerboseTestLogger.cs:13]: Or maybe we should use this: https://learn.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.testtools.unittesting.testcontext.writeline?view=visualstudiosdk-2022#microsoft-visualstudio-testtools-unittesting-testcontext-writeline

**Review by timtay-microsoft — APPROVED** [review]

**Pull request merged**